src/mock-doc/test/event.spec.ts

```diff
@@ -90,6 +90,48 @@ describe('event', () => {
     expect(ev.detail).toBe(88);
   });
 
+  it('FocusEvent() requires type', () => {
+    expect(() => {
+      // @ts-ignore checking that it throws when not supplied required arguments
+      new win.FocusEvent();
+    }).toThrow();
+  });
+
+  it('FocusEvent(type)', () => {
+    const ev = new win.FocusEvent('focus');
+    expect(ev.bubbles).toBe(false);
+    expect(ev.cancelBubble).toBe(false);
+    expect(ev.cancelable).toBe(false);
+    expect(ev.composed).toBe(false);
+    expect(ev.currentTarget).toBe(null);
+    expect(ev.defaultPrevented).toBe(false);
+    expect(ev.srcElement).toBe(null);
+    expect(ev.target).toBe(null);
+    expect(typeof ev.timeStamp).toBe('number');
+    expect(ev.type).toBe('focus');
+    expect(ev.relatedTarget).toBe(null);
+  });
+
+  it('FocusEvent(type, eventInitDict)', () => {
+    const eventInitDict = {
+      bubbles: true,
+      composed: true,
+      relatedTarget: null as EventTarget | null,
```

Comment:
might be good to have a test where we pass a legit value through for this field, what do you think?

Reply:
I'm not sure what you mean by legit value?

Reply:
sorry should have been more explicit! just something that matches the `EventTarget` interface — not a super essential test though!

Reply:
I checked into this and it doesn't look like any of the `Mock` elements (`MockInputElement`, etc) inherit from `EventTarget` currently.